backend/spec/services/extract_hubee_portail_enrollments_to_archive_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+RSpec.describe ExtractHubeePortailEnrollmentsToArchive, type: :service do
+  subject { described_class.new }
+
+  describe "#duplicated_hubee_portail_enrollments" do
+    let(:enrollment_creator) { create(:user) }
+    let(:enrollment1) { create(:enrollment, :hubee_portail, :validated, user: enrollment_creator) }
+    let(:enrollment2) { create(:enrollment, :hubee_portail, :submitted, user: enrollment_creator) }
+    let(:enrollment3) { create(:enrollment, :hubee_portail, :draft, user: enrollment_creator) }
+
+    it "returns hubee_portail enrollments which shared the same demandeur than an other hubee_portail validated enrollment" do
+      expect(subject.duplicated_hubee_portail_enrollments).to match_array([enrollment2, enrollment3])
```

Comment:
This test is returning true but it's a false positive. 

For a stronger test, in my mind, you need to create a user members of 2 differents organizations. 

then create differents enrollments with differents status and differents organizations and watch the result. 

There is some tests missing too. We need to check if the email is sent to the `demandeurs` and if the `enrollments` are archived and if the `archive events` are created too.